### PR TITLE
1.19.4

### PR DESCRIPTION
**Pull request by extremeheat**
* Rename 'session' (packet) to chat_session_update to fix auth event conflict
* Implement 1.19.4 bundle packet to process several packets at once

after https://github.com/PrismarineJS/minecraft-data/pull/703

### CODE REVIEW

File: src/client/play.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = function (client, options) {
         uuid: uuid.v4fast()
       }
 
-      client.write('session', {
+      client.write('chat_session_update', {
```
Comment (rom1504) [inline src/client/play.js:22]: why did we rename this packet ?

**Comment by rom1504** [comment]
thanks for looking into it

File: src/client/play.js (hunk 1 above)
Comment (extremeheat) [reply src/client/play.js:22]: The auth/microsoftAuth code emit the "session" event after authenticating, so it's not possible to disambiguate in event handler if it's the auth session or packet session. So I updated the naming in mcdata to match the vanilla packet name

**Comment by Maks-gaming** [comment]
when bot getting damage
image

**Comment by extremeheat** [comment]
> thanks for looking into it
> 
> please say when it's ready

Should be ready after the mc-data PR is merged. The bundle handling part can be removed if needed, I don't believe it's absolutely required

**Comment by rom1504** [comment]
thanks

**Pull request merged**